scripts/requirements.txt

```diff
@@ -1,6 +1,5 @@
 -r requirements-base.txt
 -r requirements-build-test.txt
--r requirements-doc.txt
```

Comment:
now doc requirements will be missing if you `pip install -r scripts/requirements.txt`, no?

Reply:
Yes, it is intentional. People who want to build docs will need to install doc-related dependencies using `pip install -r doc/requirements.txt`.

Reply:
Agreed, @gmarull how about we move it to `doc/` but then keep it part of the whole `requirements.txt`?

Reply:
Ya I'm with @gmarull here -- there are actually many additional non-Python reqs that people need to install anyways (Doxygen, Graphviz, ... see https://docs.zephyrproject.org/latest/contribute/documentation/generation.html#installing-the-documentation-processors) so there's no point in slowing down the initial onboarding experience IMO.